wpa_supplicant/wpa_cli_zephyr.c

```diff
@@ -78,7 +79,12 @@ static int _wpa_ctrl_command(struct wpa_ctrl *ctrl, const char *cmd, int print,
 			buf[len] = '\0';
 			if (buf[0] != '\0')
 				printf("%s", buf);
-	}
+	}else if(!print_ok){
+			buf[len] = '\0';
+			if (buf[0] != '\0')
+				printf("%s", buf);
+			print_ok = true;
```

Comment:
Is this needed now?

Reply:
This change will print only one time "OK" after flashing the board. Otherwise there is blank line.

Reply:
We can get rid of this and either print all for interactive or NONE for non-interactive.

Reply:
Okay.